apps/questapi/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "questapi",
+  "version": "1.0.0",
+  "main": "server.js",
+  "scripts": {
+    "build": "tsc -p tsconfig.json",
+    "dev": "nodemon --exec ts-node src/server.ts",
+    "start": "node src/server.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
+    "@prisma/client": "^5.22.0",
+    "fastify": "^5.1.0",
+    "fastify-plugin": "^5.0.1",
+    "mysql2": "^3.11.4",
+    "nodemon": "^3.1.7"
+  },
+  "devDependencies": {
+    "@tsconfig/node18": "^18.2.4",
+    "@types/node": "^22.9.0",
+    "@types/pino": "^7.0.4",
+    "prisma": "^5.22.0",
+    "ts-node": "^10.9.2",
```

Comment:
I heard a new package called [`tsx`](https://tsx.is/faq#how-does-tsx-compare-to-ts-node), you might want to check it out.